src/main/java/org/elmlang/intellijplugin/manualParsing/IndentationHelper.java

```diff
@@ -7,7 +7,7 @@
 import java.util.regex.Pattern;
 
 public class IndentationHelper {
-    private static final Pattern indentationPattern = Pattern.compile(".*[\r\n]([ \t][^\r\n]+)$", Pattern.DOTALL);
+    private static final Pattern indentationPattern = Pattern.compile(".*[\r\n]([^\r\n]+)$", Pattern.DOTALL);
```

Comment:
Explanation:
when the first declaration wasn't preceded by whitespace (`[ \t]`), the indentation was regarded as `0`, making the next declarations marked as not correctly indented.